mwdb/model/file.py

```diff
@@ -132,9 +193,8 @@ def iterate(self, chunk_size=1024*256):
         """
         fh = self.open()
         try:
-            if app_config.mwdb.storage_provider == StorageProviderType.S3:
-                for chunk in fh.stream(chunk_size):
-                    yield chunk
+            if hasattr(fh, "stream"):
```

Comment:
why `hasattr` instead of explicit enum check? :thinking:

Reply:
Duck typing 🦆